modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/customDataSets.spec.ts

```diff
@@ -718,47 +716,9 @@ test(
 		});
 
 		await test.step('Delete Data Set', async () => {
-			await dataSetsPage.deleteDataSet(
+			await customDataSetsPage.deleteDataSet(
 				tableSectionsWithSpecialCharactersDataSetConfig.name
 			);
 		});
 	}
 );
-
-dataSetsTabsTest(
-	'Check that there are two different tabs to navigate between Custom and System Data Sets',
-	{tag: '@LPD-37431'},
-	async ({dataSetsPage, page}) => {
-		await test.step('Navigate to Data Sets page, Custom Data Sets tab', async () => {
-			await dataSetsPage.goto();
-			await expect(
-				dataSetsPage.dataSetsEmptyState.locator('.c-empty-state-title')
-			).toContainText('No Data Sets Created');
-		});
-
-		await test.step('Create Data Set', async () => {
-			await dataSetsPage.createDataSet(tableSectionsDataSetConfig);
-		});
-
-		await assertTableColumnLabels(page);
-
-		await assertTableCellContent({
-			dataSetConfig: tableSectionsDataSetConfig,
-			page,
-		});
-
-		await assertTableActionLabels(page);
-
-		await test.step('Delete Data Set', async () => {
-			await dataSetsPage.deleteDataSet(tableSectionsDataSetConfig.name);
-		});
-
-		await test.step('Navigate to System Data Sets tab', async () => {
-			await dataSetsPage.goto('System Data Sets');
-
-			await expect(
-				dataSetsPage.dataSetsEmptyState.locator('.c-empty-state-title')
-			).toContainText('No System Data Sets Created');
-		});
-	}
-);
```

Comment:
Why do we want to delete this? I mean, is it totally covered [here](https://github.com/liferay-frontend/liferay-portal/pull/4582/files#diff-081629149553c847e5631572920a019bd27efbccc4914c5e507f00b135ded3c1R24)?

Reply:
The intent of test is fully covered with newly added test, as the new test involves navigating to `System Data Sets` tab. Check for empty FDS state is covered in multiple other tests, so I don't think it really necessary to cover it here. The content of system data sets main table is a placeholder anyway, as soon as there are real system data sets it will not be an empty state table.